Site/webpack.config.js

```diff
@@ -2,7 +2,12 @@ var configure = require('@veupathdb/site-webpack-config');
 
 const additionalConfig = {
   entry: {
-    'site-client': __dirname + '/webapp/wdkCustomization/js/client/main.js'
+    'site-client': [
```

Comment:
Future work: do we want to load the `<pdbe-molstar>` JS/CSS lazily (like we do for jQuery)?

Reply:
That's a good question. I'm not sure what is involved with lazy-loading web components. I suppose we could move the loading to a record page component, or something like that.